### DISCUSSION TITLE
Upgrading SeaweedFS from 3.63 to 3.67

### DISCUSSION THREAD

**Discussion by eliphatfs** [seaweedfs/seaweedfs#5570] [Q&A]
Is there anything need to do (e.g., running weed fix or other procedures) other than replacing the executable when upgrading seaweedfs from 3.63 to 3.67?

**Comment by marioaugustorama** [comment]
Good question, i need to do the same!

**Comment by chrislusf** [comment]
yes

**Reply by eliphatfs** [reply]
do you mean there is nothing required or we need to do something? what are they?
